packages/admin/admin/src/translator/config/ContentTranslationServiceContext.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+export interface TranslationConfig {
+    enabled: boolean;
+    translate: (input: string) => Promise<string>;
+}
+
+export const ContentTranslationServiceContext = React.createContext<TranslationConfig>({
+    enabled: false,
+    translate: async (input) => input,
```

Comment:
Yeah, i will change it to throw a Error saying that the function should never be called. If its enabled, it will always have to use the function passed to it via the Provider.